public/app/features/search/components/SectionHeader.tsx

```diff
@@ -36,10 +36,13 @@ export const SectionHeader: FC<SectionHeaderProps> = ({
     [section]
   );
 
-  return !section.hideHeader ? (
```

Comment:
This was used to hide the header in the folder view.

Reply:
Hide the header in folder view? what header? I could not see this property is set to true anywhere

Reply:
https://github.com/grafana/grafana/blob/master/public/app/features/search/hooks/useSearch.ts#L29
The header being the folder row with title. But it seems that this is redundant now since we don't render folders in the parent component for the list view.